primefaces/src/main/resources/META-INF/resources/primefaces/locales/locale-es.js

```diff
@@ -71,11 +71,11 @@ if (window.PrimeFaces) {
       pageLabel: "Página {page}",
       firstPageLabel: "Primera Página",
       lastPageLabel: "Última Página",
-      nextPageLabel: "Siguiente Página",
+      nextPageLabel: "Página Siguiente",
       previousPageLabel: "Página Anterior",
       rowsPerPageLabel: "Filas por página",
-      jumpToPageDropdownLabel: "Ir al menú desplegable de página",
-      jumpToPageInputLabel: "Ir a la entrada de página",
+      jumpToPageDropdownLabel: "Menú desplegable para ir a una página",
```

Comment:
I need to see the context of this but "Ir al menu desplegable", IMO

Reply:
Most of these are related to the datatable Paginator:
![image](https://github.com/primefaces/primefaces/assets/4399574/ddef1273-8eb3-4398-9a10-0c44fdab9f08)

Reply:
Accessibility context. The text is used by OCR (On Screen Reader).